apps/framework-docs/src/pages/moose/reference/configuration.mdx

```diff
@@ -0,0 +1,111 @@
+# Project Configuration
```

Comment:
Instead of this I would recommend to put as reference a full toml file annotated with the config. That's less to translate as a user